app/src/main/java/mozilla/lockbox/model/AccountViewModel.kt

```diff
@@ -0,0 +1,13 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package mozilla.lockbox.model
+
+data class AccountViewModel(
+    val accountName: String,
+    val displayEmailName: String,
+    val avatarFromURL: String
+)
```

Comment:
Your source model is full of optionals, so I'd make these optional too.

Then leave the choice of how to display (including the placeholder) up to the view/fragment layer.

Reply:
I'm a little confused about how to make these optional. Also struggling with the styling suggestion (`map`/`filterNotNull`/`subscribe`), I think I'm using `apply` wrong but I can't quite figure it out. Maybe we can talk on Friday or I can pull Sasha in for a little pair programming :)

Reply:
I spoke to Sasha and she was able to clarify :D